front-end/src/screens/Post/Post.tsx

```diff
@@ -1,41 +1,103 @@
+import * as moment from 'moment'
 import * as React from 'react';
+
 import ReactMarkdown from 'react-markdown';
+import { Container, Grid } from 'semantic-ui-react';
+import styled from 'styled-components'
 
 import NoPostFound from './NoPostFound';
 import RepliesBlocks from './Replies'
 import { PostAndRepliesQueryHookResult, PostFragment } from '../../generated/graphql';
+import { Tag } from '../../ui-components/Tag';
 
-const className = 'Post';
-type DataType = PostAndRepliesQueryHookResult['data']
+interface Props {
+	className?: string;
+	data: PostAndRepliesQueryHookResult['data']
+}
 
-const Post = ( { data }: {data: DataType}) => {
+const Post = ( { className, data }: Props ) => {
 	const post =  data && data.posts && data.posts[0]
 
 	return (
-		<div className={className}>
-			{ post
-				? <PostContent post={post}/>
-				: <NoPostFound/> }
-			{ post && post.replies && post.replies.length
-				? <RepliesBlocks replies={post.replies}/>
-				: null }
-		</div>
+		<Container className={className}>
+			<Grid>
```

Comment:
For another PR, we need to abstract these containers away. Repeating them in each screen is not ideal.